### PR TITLE
Update HardwareWalletTestingGuide.md

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
Add Ledger Nano X (https://twitter.com/nopara73/status/1206981498722754560).

Remove old Reddit thread (infinite loop. Redirects to GitHub, which redirects to documentation)

### CODE REVIEW

**Comment by RiccardoMasutti** [comment]
> No developer tested it, just a random dude from Twitter, so I am uncomfortable to add the fail to the docs.

Ok, I will personally test Trezor One and Model T in the next days

**Comment by MaxHillebrand** [comment]
> Ok, I will personally test Trezor One and Model T in the next days

Thanks, but these two + coldcard are heavily tested by the devs already.

File: docs/building-wasabi/HardwareWalletTestingGuide.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 | Digital BitBox |         |         |         |
 | KeepKey        |         |         |         |
 | Ledger Nano S  | pass    | pass    | pass    |
+| Ledger Nano X  | fail    | fail    | fail    |
```
Comment (MaxHillebrand) [inline docs/building-wasabi/HardwareWalletTestingGuide.md]:
```suggestion
| Ledger Nano X  |          |         |         |
```

File: docs/building-wasabi/HardwareWalletTestingGuide.md [hunk 2]
```diff
@@ -48,10 +49,8 @@ Follow this step-by-step [guide](../using-wasabi/BuildSource.md).
 
 ## Step 3: Report Results
 
-Report results on GitHub or Reddit.
+Report results on GitHub.
```
Comment (MaxHillebrand) [inline docs/building-wasabi/HardwareWalletTestingGuide.md]:
```suggestion
```

it is redundant with the next line

File: docs/building-wasabi/HardwareWalletTestingGuide.md [hunk 3]
```diff
@@ -48,10 +49,8 @@ Follow this step-by-step [guide](../using-wasabi/BuildSource.md).
...
 
 On GitHub by commenting under this pull request: [https://github.com/zkSNACKs/WalletWasabi/pull/1341](https://github.com/zkSNACKs/WalletWasabi/pull/1341)
```
Comment (MaxHillebrand) [inline docs/building-wasabi/HardwareWalletTestingGuide.md]:
```suggestion
Report the results on GitHub by commenting under [this pull request]((https://github.com/zkSNACKs/WalletWasabi/pull/1341).
```

File: docs/building-wasabi/HardwareWalletTestingGuide.md [hunk 4]
```diff
@@ -48,10 +49,7 @@ Follow this step-by-step [guide](../using-wasabi/BuildSource.md).
 
 ## Step 3: Report Results
 
-Report results on GitHub or Reddit.
```
Comment (yahiheb) [inline docs/building-wasabi/HardwareWalletTestingGuide.md]:
```suggestion
```

File: docs/building-wasabi/HardwareWalletTestingGuide.md [hunk 5]
```diff
@@ -15,6 +15,7 @@
 | Digital BitBox |         |         |         |
 | KeepKey        |         |         |         |
 | Ledger Nano S  | pass    | pass    | pass    |
+| Ledger Nano X  |          |         |         |
```
Comment (yahiheb) [inline docs/building-wasabi/HardwareWalletTestingGuide.md]:
```suggestion
| Ledger Nano X  |         |         |         |
```

File: docs/building-wasabi/HardwareWalletTestingGuide.md [hunk 6]
```diff
@@ -48,10 +49,7 @@ Follow this step-by-step [guide](../using-wasabi/BuildSource.md).
 
 ## Step 3: Report Results
 
-Report results on GitHub or Reddit.
 
-On GitHub by commenting under this pull request: [https://github.com/zkSNACKs/WalletWasabi/pull/1341](https://github.com/zkSNACKs/WalletWasabi/pull/1341)
-
-On Reddit by commenting under this thread: [https://old.reddit.com/r/WasabiWallet/comments/bdyz84/wasabi_wallet_hardware_wallet_integration_testing/](https://old.reddit.com/r/WasabiWallet/comments/bdyz84/wasabi_wallet_hardware_wallet_integration_testing/)
+Report the results on GitHub by commenting under [this pull request]((https://github.com/zkSNACKs/WalletWasabi/pull/1341).
```
Comment (yahiheb) [inline docs/building-wasabi/HardwareWalletTestingGuide.md]:
```suggestion
Report the results on GitHub by commenting under [this pull request](https://github.com/zkSNACKs/WalletWasabi/pull/1341).
```

**Pull request merged**